js/app.js

```diff
@@ -135,6 +155,16 @@ TextApp.prototype.onSettingsChanged_ = function(e, key, value) {
     case 'wraplines':
       this.editor_.setWrapLines(value);
       break;
+
+    case 'accessibilitymode':
+      this.removeEditor();
+      if (value) {
+        this.editor_ = new EditorTextArea($('#editor')[0], this.settings_);
```

Comment:
might not be worth worrying about but i just had a thought here - is it expensive to create a new editor like this? what if the user toggles this option over and over?

Reply:
It is expensive for sure, i'm hesitant to switch over to having both editors created once and switching them out because it introduces some complexity which i feel is excessive since it's unlikely this toggle will be switched often enough to cause a performance hit?

Reply:
sgtm, maybe add a code comment to that effect